utils/next.ts

```diff
@@ -54,6 +62,12 @@ export const getStaticComponentProps =
       `/${[categoryGroupName, ...paths].join("/")}`,
     )
 
+    for (const component of componentTree) {
```

Comment:
How about using `forEach` rather than `for of`?

Reply:
@illionillion `for of` loop is considered more performant than `forEach`. What is the reason for using `forEach` over `for of` loop?

Reply:
`forEach` is easier to read than `for of`.
But if performance is better, it's better to unify everything with `for of`.